mep/people/templates/people/member_card_detail.html

```diff
@@ -0,0 +1,55 @@
+{% extends 'base.html' %}
+{% load render_bundle from webpack_loader %}
+{% load static mep_tags %}
+
+{% block page-subtitle %}{{ member.firstname_last }} {{ label }} Card · Library Members · {% endblock %}
+
+{% block header %}
+{% include 'snippets/header.html' with title=member.firstname_last style="detail" %}
+{% endblock %}
+
+{% block main-style %}white{% endblock %}
+
+{% block content %}
+{% include 'snippets/breadcrumbs.html' %}
+
+<section class="card-image-viewer">
+    <div class="viewer">
+    <a rel="prev" {% if prev_card %}href="{% url 'people:member-card-detail' member.slug prev_card.short_id %}"{% else %} aria-hidden{% endif %}>
```

Comment:
since the designs don't specify the behavior of these when there isn't a next or previous, I'd rather just not render them at all - more predictable and gives an equal experience to sighted/non-sighted users. we can also just use `aria-label` to give them an accessible name; no need for inner `<span>`.

Reply:
It isn't in Zeplin but I discussed with @gissoo and she wanted it visible but grayed out (that's also consistent with what we have elsewhere). You can discuss further with her if you disagree.  I wasn't sure if we were going to make the pagination ajaxy or not; if we do, won't we have the same problem with the member page for updating the pagination?

Using aria-label sounds much better than an inner span!